README.md

```diff
@@ -1,33 +1,37 @@
 # Welcome
 
-This repository contains the source code for:
+This repository contains the source code & documentation for:
 
 * [Windows Terminal](https://www.microsoft.com/en-us/p/windows-terminal-preview/9n0dx20hk701)
 * The Windows console host (`conhost.exe`)
 * Components shared between the two projects
 * [ColorTool](https://github.com/Microsoft/Terminal/tree/master/src/tools/ColorTool)
 * [Sample projects](https://github.com/Microsoft/Terminal/tree/master/samples) that show how to consume the Windows Console APIs
-  
+
 Other related repositories include:
 
 * [Console API Documentation](https://github.com/MicrosoftDocs/Console-Docs)
 
 ## Installing & running Windows Terminal
 
-> 👉 Note: To run the Windows Terminal you'll need to be running Windows 10 1903 (build 18362) or later
+### Operating System Support
+
+Windows Terminal can be installed on Windows 10 1903 (build 18362) or later.
 
-### From the Microsoft Store
+### Install via the Microsoft Store
 
-Download the Microsoft Terminal free from the Microsoft Store and it'll be continuously updated:
+We recommend downloading the Windows Terminal from the Microsoft Store and automatically receive upgrades when we release new builds:
```

Comment:
```suggestion
We recommend downloading the Windows Terminal from the Microsoft Store so it will automatically receive upgrades when we release new builds:
```